src/WebJobs.Script.WebHost/Handlers/WebScriptHostHandler.cs

```diff
@@ -13,7 +12,7 @@ namespace Microsoft.Azure.WebJobs.Script.WebHost.Handlers
     public class WebScriptHostHandler : DelegatingHandler
     {
         public const int HostTimeoutSeconds = 30;
-        public const int HostPollingIntervalMilliseconds = 500;
+        public const int HostPollingIntervalMilliseconds = 25;
```

Comment:
is the faster polling intentional here?

Reply:
Yes. Because the CanInvoke check is fast (just checking some in memory state) we can improve coldstart by reducing this.